### PR TITLE
Fix indent

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @EPashkin @sdroege

### CODE REVIEW

**Comment by sdroege** [comment]
👍

**Comment by EPashkin** [comment]
👍

**Comment by EPashkin** [comment]
Wait

File: atk-sys/src/lib.rs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// This file was generated by gir (ac9a8da) from gir-files (0bcaef9)
+// This file was generated by gir (a4dcfea) from gir-files (71d73f0)
```
Comment (EPashkin) [inline atk-sys/src/lib.rs]: Updated with not last gir-files

**Comment by EPashkin** [comment]
@sdroege you will add `StaticType` support for unions?
If yes, this PR better wait for this.

**Comment by GuillaumeGomez** [comment]
Oh. :'(

**Comment by sdroege** [comment]
I'll check later. If that doesn't require more changes elsewhere, sure.

**Comment by sdroege** [comment]
Not today anymore, sorry

**Comment by GuillaumeGomez** [comment]
No problem, I merge then! :D

**Pull request merged**